packages/cli/react-native.config.js

```diff
@@ -148,6 +148,12 @@ module.exports = {
           name: "--source-map-file <file>",
           description: "The sourceMap file to generate licence contents for.",
         },
+        {
+          name: "--json",
+          description: "Output license information as a JSON",
+          default: true,
```

Comment:
IIRC the default output is plain text... if I'm wrong, ignore this comment. 

Otherwise, this seems dangerous, as you're changing the defaults and callers probably won't know that their TPN file has changed formats as it tends to get generated and packaged/nuget-ed, rather than checked in.
```suggestion
          default: false,
```

Reply:
Good catch! I made a copy/pasta mistake 😛